src/components/ChallengeEditor/ChallengeViewTabs/index.js

```diff
@@ -85,17 +86,19 @@ const ChallengeViewTabs = ({
       <div className={styles.topContainer}>
         <div className={styles.leftContainer}>
           <div className={styles.title}>{challenge.name}</div>
-          {!isTask && (
-            <div
-              className={cn(
-                styles.actionButtons,
-                styles.button,
-                styles.actionButtonsLeft
-              )}
-            >
-              <LegacyLinks challenge={challenge} challengeView />
-            </div>
-          )}
+
+          <div
+            className={cn(
+              styles.actionButtons,
+              styles.button,
+              styles.actionButtonsLeft
+            )}
+          >
+            { isTask ? (<ForumLink challenge={challenge} />)
```

Comment:
@dedywahyudi this logic would not render the forum link for non task challenges, is it what intended here?

Reply:
@vikasrohit 
{ isTask ? (<ForumLink challenge={challenge} />) : (<LegacyLinks challenge={challenge} challengeView />) }

So, Non Task challenge render by LegacyLinks component already. But, forum link use new ForumLink component